app/lib/actions/pulp3/abstract.rb

```diff
@@ -2,6 +2,7 @@ module Actions
   module Pulp3
     class Abstract < Actions::Base
       middleware.use ::Actions::Middleware::RemoteAction
+      middleware.use Actions::Middleware::PulpServicesCheck
```

Comment:
I think @sjha4 might be saying to create a new middleware that checks pulp3 only:

https://github.com/Katello/katello/blob/a1911c7b68208c54d0f8667dfbc12a17bebf240c/app/lib/actions/middleware/pulp_services_check.rb#L5